### DISCUSSION TITLE
Adjust spine logo

### DISCUSSION THREAD

**Discussion by fb-correia** [joaomlourenco/novathesis#320] [Q&A] [answered]
Hello, 
Is there any way do adjust the spine logo? I have the code for spine=trim but it isn't working.

Thank you

![image](https://user-images.githubusercontent.com/98908443/160092015-7277b52f-269f-45c4-ab44-6f48820a01be.png)
![image](https://user-images.githubusercontent.com/98908443/160092107-1d595605-d0c7-458a-8621-e4bceb4fd10f.png)

**Comment by joaomlourenco** [comment]
That shouldn't be happening. 

Please tell me which version you are using and the number of the last page in your document.

**Reply by fb-correia** [reply]
I'm using [6.9.10] version and the last page is 115

**Comment by joaomlourenco** [comment, marked as answer]
Thanks for raising the problem. That is definitely a bug.  

To fix it, in line 61 of file `NOVAthesisFiles/Schools/nova/fct/nova-fct-defaults.ldf`
Replace
```latex
\spine(logo,msc,scale)={1.5}
```
by
```latex
\spine(logo,msc,scale)={1}
```

**Reply by fb-correia** [reply]
Solved! Thank you
